src/bloaty.cc

```diff
@@ -291,6 +291,16 @@ std::string NameMunger::Munge(string_view name) const {
   return name_str;
 }
 
+struct Label {
+  Label(const std::string& name) : name(name) {}
+  Label(const std::string& name, uint64_t vm_capacity, uint64_t file_capacity)
+      : name(name), vm_capacity(vm_capacity), file_capacity(file_capacity) {}
+
+  std::string name;
+  uint64_t vm_capacity = 0;
```

Comment:
Please add mention of the units (bytes). If there is special meaning for some values (e.g.0) please document.